covid_graphs/covid_graphs/country_graph.py

```diff
@@ -44,59 +64,75 @@ def __init__(
         self.short_name = report.short_name
         self.long_name = report.long_name
 
-        # TODO(lukas): Figure out better strategy for more predictions
         if len(country_predictions) >= 1:
-            self.prediction_date = country_predictions[0].prediction_event.date
+            self.prediction_date = max(
+                country_prediction.prediction_event.date
+                for country_prediction in country_predictions
+            )
```

Comment:
Note: This can stay as is. I think though that `self.prediction_date` is an ill-defined concept. If we have more than one prediction date we have to come up with a new UI to be able to display them per trace.

Actually, I'd move this logic just before plotting the green zone, there is no need to compute it at init.

Reply:
Yes, we don't have a good UI for this, but taking a maximum is a much better choice.

The point of this refactoring is to do all computation before the UI is drawn, so that it can be also reused in the JS web. We should store it here, or the whole predictions.

Reply:
I would argue that UI display should get all the prediction dates, otherwise we are losing information. (It is a minor comment, I'm fine with not doing that.)

Reply:
Sure, but that's a refactor that I don't want to do right now :) I just wanted to refactor and keep the same functionality.